SPECS/kernel-hyperv/config

```diff
@@ -227,6 +227,7 @@ CONFIG_ARCH_WANT_DEFAULT_BPF_JIT=y
 CONFIG_BPF_JIT_DEFAULT_ON=y
 # CONFIG_USERFAULTFD is not set
 CONFIG_ARCH_HAS_MEMBARRIER_SYNC_CORE=y
+CONFIG_KCMP=y
```

Comment:
Should we mention this config change in the changelog as well? Just so that it's easier to find when this was introduced?

Reply:
This particular type of config addition/removal happens frequently with every kernel update. And this value is a default setting applied based on the values of other configs we use. I don't think the changelog needs a specific entry about it.